File: DoAnLau-API/DoAnLau-API/DoAnLau-API/Models/MenuCategory.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace DoAnLau_API.Models
{
    public class MenuCategory
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
       
        public string menuCategory_Id {  get; set; }


        [Column(TypeName = "nvarchar(100)")]
        public string categoryName { get; set; }

        [Column(TypeName = "bit")]
        public bool state { get; set; }

        public ICollection<Menu> menus { get; set; }
    }
}

File: file-angular/src/app/Models/MenuCategory.ts
export class MenuCategory {
  menuCategory_Id: string;
  categoryName: string;
  state: boolean;

  constructor(menuCategory_Id: string, categoryName: string, state: boolean) {
    this.menuCategory_Id = menuCategory_Id;
    this.categoryName = categoryName;
    this.state = state;
  }
}
